IDEAS/Buildings/Components/Interfaces/PartialSurface.mo

```diff
@@ -30,6 +30,10 @@ partial model PartialSurface "Partial model for building envelope component"
   parameter Modelica.Units.SI.Temperature TRef_a=291.15
     "Reference temperature of zone on side of propsBus_a, for calculation of design heat loss"
     annotation (Dialog(group="Design power", tab="Advanced"));
```

Comment:
This parameter is also claimed to be used to calculate design heat losses. Can't we use this existing one then? This approach was already applied to the `InternalWall`.